CADability/ShowPropertyLine.cs

```diff
@@ -49,7 +49,8 @@ public ShowPropertyLine(Line line, IFrame frame): base(frame)
 
             directionProperty = new GeoVectorProperty("Line.Direction", Frame, false);
             directionProperty.IsNormedVector = true;
-            directionProperty.GetGeoVectorEvent += new CADability.UserInterface.GeoVectorProperty.GetGeoVectorDelegate(OnGetDirection);
+			directionProperty.IsAngle = true;
+			directionProperty.GetGeoVectorEvent += new CADability.UserInterface.GeoVectorProperty.GetGeoVectorDelegate(OnGetDirection);
             directionProperty.SetGeoVectorEvent += new CADability.UserInterface.GeoVectorProperty.SetGeoVectorDelegate(OnSetDirection);
```

Comment:
You used tabs instead of spaces.

```suggestion
            directionProperty.IsAngle = true;
            directionProperty.GetGeoVectorEvent += new CADability.UserInterface.GeoVectorProperty.GetGeoVectorDelegate(OnGetDirection);
            directionProperty.SetGeoVectorEvent += new CADability.UserInterface.GeoVectorProperty.SetGeoVectorDelegate(OnSetDirection);
```